build.gradle.kts

```diff
@@ -9,7 +9,7 @@ version = "1.5.2"
 repositories {
     maven {
         name = "TerraformersMC"
-        url = uri("https://maven.quiltmc.org/repository/release/")
+        url = uri("https://maven.terraformersmc.com")
```

Comment:
Shouldn't it be `https://maven.terraformersmc.com/releases/`

Reply:
If not specified it shows `releases/`, as seen in the web index https://maven.terraformersmc.com/ and https://maven.terraformersmc.com/com/terraformersmc/modmenu/2.0.13/maven-metadata.xml
I actually didn't notice that it was in a releases directory, but no other directory is used